File: MT/LMS.Core/ViewModel/AttendanceVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LMS.Core.ViewModel
{
    public class AttendanceVM : BaseViewModel
    {

        public string? UserId { get; set; }
        public string? User { get; set; }
        public string? InTime { get; set; }
        public string? OutTime { get; set; }
        public string? ScheduleTime { get; set; }
        public string? WorkedHours { get; set; }
        public DateTime? Date { get; set; }

        //public string Schedule { get; set; }
    }
}

File: Back Office/src/app/views/att/Models/AttendanceVM.ts
export class AttendanceVM {
    id!: number
    userId?: string
    user?: string
    inTime?: string
    outTime?: string        
    workedHours? :string
    ScheduleTime? :string
    // date? :  Date = new Date 
    from? :  Date = new Date 
    to? :  Date   = new Date
    isActive?: boolean
}
